### DISCUSSION TITLE
Thorough search for electricity meter

### DISCUSSION THREAD

**Discussion by Nils154** [bemasher/rtlamr#174] [Q&A]
This is an absolutely amazing application! I just started it with RTLAMR without any options on my RPi2 and it found both my water meter (R900) and my gas meter (SCM). Now I am wondering if my electric meter is compatible or not. How do I do a thorough search. Instruction say to try a couple of frequencies, how is that thorough? Looking for guidance on a thorough search.

Thanks!

**Comment by bemasher** [comment]
You'll want to physically inspect your meter first. Knowing the FCC ID also helps.

**Comment by Nils154** [comment]
Ah! Yes. No FCC, all is has is a sticker with 
‘PowerLAN address’.

Concluding that that means it does some sort of ethernet over the power lines to get the data and not radio. Bummer, as I do get my gasmeter (SCM) and watermeter (R900) 

--
Nils Gokemeijer (Sent from iPhone)
